### PR TITLE
add DepsDataSource

### PR DESCRIPTION
**Pull request by keshav-space**
- add Deps DataSource
- add tests for Deps DataSource

### CODE REVIEW

**Comment by TG1999** [comment]
Add doctsrings for all the functions and also add doctests/ unit tests for functions which have not been tested at all

File: vulntotal/datasources/deps.py [hunk 1]
```diff
@@ -0,0 +1,122 @@
+#
+# Copyright (c) nexB Inc. and others. All rights reserved.
+# http://nexb.com and https://github.com/nexB/vulnerablecode/
+# The VulnTotal software is licensed under the Apache License version 2.0.
```
Comment (pombredanne) [inline vulntotal/datasources/deps.py:4]: Same nit as elsewhere, please use the new simpler header

Comment (pombredanne) [reply vulntotal/datasources/deps.py]: Fixed in https://github.com/nexB/vulnerablecode/pull/789/commits/84100766a5f6be7a4183ee7c6f165cc1924b3123

**Review by pombredanne — APPROVED** [review]
LGTM .... Merging! :+1:

**Pull request merged**